src/main/java/teammates/ui/pagedata/PageData.java

```diff
@@ -681,56 +679,6 @@ public InstructorFeedbackSessionActions getInstructorFeedbackSessionActions(Feed
     }
 
     /**
-     * Returns the type of people that can view the comment.
-     */
-    public String getTypeOfPeopleCanViewComment(CommentAttributes comment) {
-        StringBuilder peopleCanView = new StringBuilder(100);
-        for (int i = 0; i < comment.showCommentTo.size(); i++) {
-            CommentParticipantType commentViewer = comment.showCommentTo.get(i);
-            if (i == comment.showCommentTo.size() - 1 && comment.showCommentTo.size() > 1) {
-                peopleCanView.append("and ");
-            }
-
-            switch (commentViewer) {
-            case PERSON:
-                peopleCanView.append("recipient, ");
-                break;
-            case TEAM:
-                if (comment.recipientType == CommentParticipantType.TEAM) {
-                    peopleCanView.append("recipient team, ");
-                } else {
-                    peopleCanView.append("recipient's team, ");
-                }
-                break;
-            case SECTION:
-                if (comment.recipientType == CommentParticipantType.SECTION) {
-                    peopleCanView.append("recipient section, ");
-                } else {
-                    peopleCanView.append("recipient's section, ");
-                }
-                break;
-            case COURSE:
-                if (comment.recipientType == CommentParticipantType.COURSE) {
-                    peopleCanView.append("the whole class, ");
-                } else {
-                    peopleCanView.append("other students in this course, ");
-                }
-                break;
-            case INSTRUCTOR:
-                peopleCanView.append("instructors, ");
-                break;
-            default:
-                break;
-            }
-        }
-        String peopleCanViewString = peopleCanView.toString();
-        if (peopleCanViewString.isEmpty()) {
-            return peopleCanViewString;
-        }
-        return removeEndComma(peopleCanViewString);
-    }
-
-    /**
```

Comment:
Seems fine, due to the diff being shown on GitHub this line is shown as removed, but old line 683 is still present.